artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java

```diff
@@ -554,10 +554,12 @@ public boolean updateMember(long uniqueEventID, String nodeId, TopologyMemberImp
     * @return
     */
    @Override
-   public boolean removeMember(final long uniqueEventID, final String nodeId) {
+   public boolean removeMember(final long uniqueEventID, final String nodeId, final boolean disconnect) {
       if (nodeId.equals(nodeManager.getNodeId().toString())) {
-         ActiveMQServerLogger.LOGGER.possibleSplitBrain(nodeId);
-         return false;
+         if (!disconnect) {
+            ActiveMQServerLogger.LOGGER.possibleSplitBrain(nodeId);
+            return false;
+         }
```

Comment:
With my other comment about the 'info message if disconnect' you added previously, I was meaning more to leave it always logging the warning, wheres now this is back to not saying anything.

Unless we know for sure there isnt such possibility because it was a shutdown?  Its not clear to me, I dont know enough about the clustering to say either way. 

This also seems to change it from 'rejecting a topology update' to accepting it in this case, which seems a larger change than just the logging, and which I'm similarly not clear on the effect of. Need others with more knowledge here...@clebertsuconic @jbertram ?

Reply:
yes maybe I should leave it to return false.
Leaving it to WARN will mean having a warning when a node leaves which seems strange to me